src/common/scoped_message_writer.h

```diff
@@ -91,7 +91,7 @@ class scoped_message_writer
     {
       m_flush = false;
 
-      MCLOG(m_log_level, "msgwriter", m_oss.str());
+      MCLOG_FILE(m_log_level, "msgwriter", m_oss.str());
 
       if (epee::console_color_default == m_color)
```

Comment:
Would it be better to just remove these lines instead? Writing directly to `std::cout` without a lock, etc.